src/main/java/org/zalando/nakadi/controller/SubscriptionController.java

```diff
@@ -143,4 +148,35 @@ private Subscription getExistingSubscription(final SubscriptionBase subscription
                 subscriptionBase.getConsumerGroup());
     }
 
+    private final class EventTypesHolder {
```

Comment:
Yes, of course you need the second list. What I meant is that you don't need to return two lists from the method. Because the only reason of EventTypesHolder to exist is that you want to return two lists from the method. This class is used only once only in this place and will never be used again. 
That's why I think the more clear way would be to write a method that returns only existing event-types and then you find missing event-types by comparing to lists (all - existing).